composer.json

```diff
@@ -10,20 +10,45 @@
             "homepage": "https://github.com/lmc-eu"
         }
     ],
+    "autoload": {
+        "psr-4": {
+            "Lmc\\CodingStandard\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Lmc\\CodingStandard\\": "tests/"
+        }
+    },
     "require": {
         "symplify/easy-coding-standard": "^4.0.2"
     },
     "require-dev": {
         "j13k/yaml-lint": "^1.1",
-        "mhujer/yaml-sort-checker": "^1.2"
+        "mhujer/yaml-sort-checker": "^1.2",
```

Comment:
Did you consider using `lmc/coding-standards` here as well?

Reply:
Why :)? This is the package itself :upside_down_face: